Makefile

```
@@ -1,9 +1,14 @@
-all: specs/index.html
+RELEASE_DIR:=out
+
+build: specs/index.html
+	mkdir -p ${RELEASE_DIR}
+	cp -r $< specs/diagrams ${RELEASE_DIR}/
+	cp -r TR ${RELEASE_DIR}/
 
 specs/index.html: specs/index.bs
 	bikeshed spec $< $@
 
 serve:
 	cd specs && bikeshed serve
 
-.PHONY: serve
\ No newline at end of file
+.PHONY: serve release
```

I believe this should now be `build`, no?

```suggestion
.PHONY: serve build
```